client/src/plus/deep-dives/overview/index.scss

```diff
@@ -6,196 +6,167 @@
-  overflow: hidden;
-  padding: 0;
-  position: relative;
+.deep-dives-overview {
+  background-color: theme.$white;
+  margin: 0 auto;
+  max-width: 600px;
 
   &::before {
-    background: transparent url("./assets/hero.svg") center center no-repeat;
+    background: transparent url("./assets/deep-dive-hero.svg") center center
+      no-repeat;
     background-size: cover;
     content: "";
     display: block;
-    height: 100%;
-    min-height: 365px;
-    position: absolute;
-    width: 100%;
-  }
-
-  .header-wrapper {
-    background-color: rgba(0, 0, 0, 0.4);
-    height: 100%;
     margin: auto;
     max-width: layout.$max-width-default;
+    min-height: 212px;
     padding: (layout.$base-spacing * 2) layout.$base-spacing;
     position: relative;
     z-index: layout.$bottom-layer;
+  }
+}
 
-    @media #{layout.$mq-small-desktop-and-up} {
-      background-color: transparent;
-      display: grid;
-      gap: 0 math.div(layout.$base-spacing, 2);
-      gap: 0 (layout.$base-spacing * 2.5);
-      grid-template-columns: 1fr 0;
-      padding: 120px;
+.deep-dive-toc,
+.overview-hero {
+  a {
+    &:link,
+    &:visited {
+      color: theme.$mdn-neutral900;
+      text-decoration: none;
+    }
+
+    &:hover,
+    &:focus {
+      text-decoration: underline;
     }
   }
+}
+
+.overview-hero {
+  padding: 0 layout.$base-spacing math.div(layout.$base-spacing, 2);
+
+  @media #{layout.$mq-large-desktop-and-up} {
+    margin: 36px auto layout.$base-spacing;
+    max-width: layout.$max-width-default;
+  }
 
   .header-content {
-    max-width: 550px;
+    margin-top: -36px;
     position: relative;
     z-index: layout.$top-layer;
 
-    h1 {
-      font-size: typography.$xxl-font-size;
-      line-height: typography.$heading-line-height;
-
-      margin-block-end: 0.67em;
-      margin-block-start: 0.67em;
-      margin-inline-end: 0;
-      margin-inline-start: 0;
-
-      @media #{layout.$mq-small-desktop-and-up} {
-        font-size: theme.$marketing-hero-h1-font-size;
-        margin: layout.$base-spacing 0;
-      }
+    @media #{layout.$mq-small-desktop-and-up} {
+      margin-top: -24px;
     }
 
-    h2 {
-      background: theme.$gradient-accent;
-      background-clip: text;
-      color: theme.$mdn-green400;
-      font-size: 18px;
-      line-height: 160%;
-      margin-top: 0;
-      text-transform: uppercase;
+    @media #{layout.$mq-large-desktop-and-up} {
+      margin-top: -56px;
     }
 
-    a {
-      &:link,
-      &:visited {
-        color: theme.$white;
-        text-decoration: none;
-      }
-
-      &:hover,
-      &:focus {
-        text-decoration: underline;
-      }
-    }
-
-    p {
-      font-size: typography.$default-font-size;
-      margin-bottom: 0;
-      max-width: 450px;
-
-      @media #{layout.$mq-small-desktop-and-up} {
-        font-size: typography.$small-medium-font-size;
-      }
+    h1 {
+      font-size: 36px;
+      line-height: typography.$heading-line-height;
```

Comment:
Line height is 100% now: https://www.figma.com/file/7lbOm57iOjSVIGORzXqrlW/MDN-Articles?node-id=276%3A3152